docs/docs/getting-started/first-steps-local.md

```diff
@@ -4,19 +4,25 @@
 With the `constellation mini` command, you can deploy and test Constellation locally without a cloud subscription. This mode is called MiniConstellation. Conceptually, MiniConstellation is similar to [MicroK8s](https://microk8s.io/), [K3s](https://k3s.io/), and [minikube](https://minikube.sigs.k8s.io/docs/).
```

Comment:
regarding
https://github.com/edgelesssys/constellation/blob/c52dfb09b903fd1b9e241580c7b0f2e57357b2e3/cli/internal/cmd/miniup.go#L101-L102
how about extending the message such that users get to know the platform which works without needing to consult the docs?

Reply:
Should we directly refer to the linux/amd64 platform here? Is a change of the supported platforms in the near future likely?

Reply:
Not sure. @daniel-weisse do you have an opinion on this?

Reply:
Change in support is unlikely, since the miniConstellation requires libvirt with QEMU/KVM, which is not available on Windows, and has poor performance and does not support the features we need on OSX

Reply:
https://github.com/edgelesssys/constellation/blob/46a13eb4f76af7fcbeaa19685ad08184ec3e3571/cli/internal/cmd/miniup.go#L101-L103

I did it like that now, what do you think? Was kind of unsure on how to phrase it